pkgs/servers/web-apps/peering-manager/default.nix

```diff
@@ -7,31 +7,7 @@
 , plugins ? ps: []
 }:
 
-let
-  py = python3.override {
```

Comment:
Also `peering-manager` is not `netbox`, this is confusing to have it in the same commit.